client/app.jsx

```diff
@@ -0,0 +1,153 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import Calendar from './components/Calendar.jsx';
+import rightArrow from './styles/icons/right_arrow.svg';
+import downArrow from './styles/icons/down_arrow.svg';
+import flag from './styles/icons/flag.svg';
+import question from './styles/icons/question.svg';
+import './styles/style.css';
+import $ from 'jquery';
+
+class App extends React.Component {
+  constructor() {
+    super();
+    this.today = new Date();
+    this.state = {
+      year: this.today.getFullYear(),
+      month: this.today.getMonth(),
+      requestedDates: [],
+      stage: 0,
+      price: 0,
+      cleaning: 0,
+      maxGuests: 0,
+      minStay: 0,
+      serviceFee: 0,
+      bookedDates: []
+    };
+  }
+
+  componentDidMount() {
+    $.get('/cal', (res) => {
+      let result = JSON.parse(res);
+      this.setState({
+        price: result.price,
+        cleaning: result.cleaning,
+        maxGuests: result.maxGuests,
+        minStay: result.minStay,
+        serviceFee: result.serviceFee,
+        bookedDates: result.year
+      });
+    });
+  }
+
+  handleCheckInClick(newStage) {
```

Comment:
I could be wrong, I think in general 'handle' is used to identify functions that take an event as an argument. In cases where you're just taking a string or number value, I would suggest something a little more directly related to the purpose, like 'setStage'.